internal/service/advancedclustertpf/model_ClusterDescription20240805.go

```diff
@@ -25,13 +25,18 @@ type LegacySchemaInfo struct {
 	RootDiskSize               *float64
 }
 
-func NewTFModel(ctx context.Context, input *admin.ClusterDescription20240805, timeout timeouts.Value, diags *diag.Diagnostics, legacyInfo *LegacySchemaInfo) *TFModel {
+type ExtraAPIInfo struct {
```

Comment:
Although we could define only `containerIDs map[string]string` as a parameter, I think this is more "future-proof".
(Couldn't use LegacySchemaInfo, as that is only meant for numShards > 1 configurations)

Reply:
i have mixed feeling about anticipating future needs as it can create overcomplexity but it's ok here.